### PR TITLE
WIP: MGMT-7340: add labels to the api

### PR DESCRIPTION
**Pull request by pkliczewski**
This PR applies labels to freshly installed node base on labels available in the host table.

Signed-off-by: Piotr Kliczewski <piotr.kliczewski@gmail.com>

### CODE REVIEW

**Comment by pkliczewski** [comment]
This PR depends on https://github.com/openshift/assisted-service/pull/2320

**Comment by tsorya** [comment]
@pkliczewski we still need it?

**Comment by pkliczewski** [comment]
@tsorya yes, this work will be continued by @rewantsoni

File: src/k8s_client/k8s_client.go [hunk 1]
```diff
@@ -159,6 +160,12 @@ func (c *k8sClient) ListMasterNodes() (*v1.NodeList, error) {
 	return nodes, nil
 }
 
+func (c *k8sClient) PatchNodeLabels(node *v1.Node, labels string) error {
```
Comment (dvachhan) [inline src/k8s_client/k8s_client.go:163]: @pkliczewski Is this where the labels get applied to the nodes?

**Pull request closed without merging**